docs/getting_started.md

```diff
@@ -74,7 +74,7 @@ The initial password is autogenerated to be the pod name of the
 Argo CD API server. This can be retrieved with the command:
 
 ```bash
-kubectl get pods -n argocd -l app.kubernetes.io/name=argocd-server -o name | cut -d'/' -f 2
```

Comment:
Instructions are correct. This paragraph explains how to get the password for v1.8 earlier and then the new process for v1.9+. May be move new process to the top ?

Reply:
@jannfis what do you think?